FrostBlossomMC/src/main/java/de/alaust/frostblossommc/item/consumable/ConsumeListener.kt

```diff
@@ -0,0 +1,20 @@
+package de.alaust.frostblossommc.item.consumable
+
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.event.player.PlayerItemConsumeEvent
+
+class ConsumeListener : Listener {
+    companion object {
+        private val consumeHandler = ArrayList<Consumable>()
+
+        fun registerConsumeHandler(consumable: Consumable) = consumeHandler.add(consumable)
```

Comment:
Maybe it makes sense to also have a `unregisterConsumeHandler` function, but this requires more changes than just in this file.

Reply:
There is no use case where an unregister function is needed. Also the developer which uses this framework shouldn't register and unregister those handlers by itself because this could easily result in inconsistency / bugs in their code. Therefore I made it harder to access this logic.